v22.1/cockroach-start-single-node.md

```diff
@@ -247,7 +247,7 @@ Scaling a cluster started with `cockroach start-single-node` involves restarting
 	server drained and shutdown completed
 	~~~
 
-2. Restart the node with the [`cockroach start`](cockroach-start.html) command:  
+2. Restart the node with the [`cockroach start`](cockroach-start.html) command:
```

Comment:
For future reference, our internal style is to use `1.` for each item in an ordered list: https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#lists

I usually fix this opportunistically, but it's obviously a low priority. It helps prevent confusion where tutorial steps are likely to be updated.